### DISCUSSION TITLE
Ulid key

### DISCUSSION THREAD

**Discussion by Grazulex** [filamentphp/filament#6297] [Help] [answered]
Hello, 
I'm using Ulid like id in my models.
When I try to create the ressources files for my orders, I have this error : 

`Missing required parameter for [Route: filament.resources.users.edit] [URI: admin/users/{record}/edit] [Missing parameter: record].`

How can I used Ulid with Filament ?

Thanks

**Comment by danharrin** [comment, marked as answer]
Have you set the route key name in your model correctly?
